plugins/db-sqlserver/webpack.config.js

```diff
@@ -0,0 +1,45 @@
+const path = require("path");
+const webpack = require("webpack");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+
+/** @type {import("webpack").Configuration} */
+module.exports = {
+  mode: "production",
+  target: "node",
+  entry: "./src/index.ts",
+  plugins: [
+    new webpack.SourceMapDevToolPlugin({
+      filename: "[name].js.map",
+    }),
+    new CopyWebpackPlugin({
+      patterns: [
+        { from: "src/static", to: "static", noErrorOnMissing: true },
+        { from: "src/templates", to: "templates", noErrorOnMissing: true },
+      ],
+    }),
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: "ts-loader",
+        exclude: /node_modules/,
+      },
+    ],
+  },
+  resolve: {
+    extensions: [".ts", ".js"],
+  },
+  optimization: {
+    minimize: false,
+  },
+  output: {
+    filename: "index.js",
+    path: path.resolve(__dirname, "dist"),
+    libraryTarget: "commonjs2",
+    clean: true,
+  },
+  externals: {
+    "prisma-schema-dsl": "prisma-schema-dsl",
```

Comment:
This probably need to be added as peerDependency but we can handle it in a different PR as we always have `prisma-schema-dsl` dependency in the `data-service-generator`